bedrock/firefox/views.py

```diff
@@ -619,3 +619,7 @@ def get_template_names(self):
             template = 'firefox/feedback/unhappy.html'
 
         return [template]
+
+
+class TrackingProtectionTourView(TemplateView):
```

Comment:
This will not use our l10n machinery. Need to import `LangFilesMixin` from `lib.l10n_utils` and add it to the inheritance for this view. [Example here](https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/views.py#L157).